ansible/roles/operator-pipeline/templates/openshift/tasks/preflight-trigger.yml

```diff
@@ -46,188 +46,166 @@ spec:
       name: result_output_file
     - description: ''
       name: artifacts_output_dir
-  volumes:
-    - name: kubeconfig-volume
-      secret:
-        secretName: "$(params.prow_kubeconfig_secret_name)"
-    - name: gpg-key-volume
-      secret:
-        secretName: "$(params.gpg_decryption_key_secret_name)"
```

Comment:
I don't think we can do away with these. I'm seeing a volumeMount reference in your changes that specifically calls out at least the gpg-key-volume